### PR TITLE
Fixed phpdoc

### PR DESCRIPTION
**Pull request by core23**

### CODE REVIEW

File: Admin/BlockAdmin.php [hunk 1]
```diff
@@ -23,8 +23,14 @@
  */
 class BlockAdmin extends BaseBlockAdmin
 {
+    /**
+     * {@inheritdoc)
+     */
     protected $parentAssociationMapping = 'page';
 
+    /**
+     * {@inheritdoc)
```
Comment (Soullivaneuh) [inline Admin/BlockAdmin.php]: `)` => `}`

File: Admin/BlockAdmin.php [hunk 2]
```diff
@@ -23,8 +23,14 @@
  */
 class BlockAdmin extends BaseBlockAdmin
 {
+    /**
+     * {@inheritdoc)
```
Comment (Soullivaneuh) [inline Admin/BlockAdmin.php]: `)` => `}`

File: Admin/PageAdmin.php [hunk 3]
```diff
@@ -48,6 +49,9 @@ class PageAdmin extends Admin
      */
     protected $cacheManager;
 
+    /**
+     * {@inheritdoc)
```
Comment (Soullivaneuh) [inline Admin/PageAdmin.php]: `)` => `}`

File: Admin/PageAdmin.php [hunk 4]
```diff
@@ -441,7 +445,7 @@ public function setSiteManager(SiteManagerInterface $siteManager)
     }
 
     /**
-     * @return array
+     * @return SiteInterface
```
Comment (Soullivaneuh) [inline Admin/PageAdmin.php]: Sure about that? Should not be `SiteInterface[]`?

File: Admin/SharedBlockAdmin.php [hunk 5]
```diff
@@ -88,7 +91,7 @@ protected function configureFormFields(FormMapper $formMapper)
     }
 
     /**
-     * {@inheritDoc).
+     * {@inheritdoc)
```
Comment (Soullivaneuh) [inline Admin/SharedBlockAdmin.php]: `)` => `}`

**Comment by core23** [comment]
@Soullivaneuh thanks for your review :+1:

**Comment by Soullivaneuh** [comment]
Thank for your contribution. :+1:

**Pull request merged**